src/redux/books/books.js

```diff
@@ -0,0 +1,61 @@
+import axios from 'axios';
+import uuid from 'react-uuid';
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+/* eslint-disable no-console */
+// to add action types
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+const READ_BOOKS = 'bookstore/books/READ_BOOK';
+
+// to get the url
+const apiUrl = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/EHbt3DaBZgT4pqfPpOLB/books/';
+
+// the get book action
+export const getBookThunk = createAsyncThunk(READ_BOOKS, async () => {
```

Comment:
`createAsyncThunk` is used in the project for creating thunk action creators 🚀 Well done!